clang/test/Preprocessor/riscv-target-features.c

```diff
@@ -1612,6 +1613,14 @@
 // RUN:   -o - | FileCheck --check-prefix=CHECK-SUPM-EXT %s
 // CHECK-SUPM-EXT: __riscv_supm 8000{{$}}
 
+// RUN: %clang --target=riscv32 -menable-experimental-extensions \
+// RUN:   -march=rv32i_ssqosid1p0 -E -dM %s \
+// RUN:   -o - | FileCheck --check-prefix=CHECK-SSQOSID-EXT %s
+// RUN: %clang --target=riscv64 \
+// RUN:   -march=rv64i_ssqosid1p0 -E -dM %s -menable-experimental-extensions \
```

Comment:
Why is `-menable-experimental-extensions` here?

Reply:
It's not ratified yet right?

Reply:
It is required for clang when using experimental RISC-V extensions.
> def menable_experimental_extensions : Flag<["-"], "menable-experimental-extensions">, Group<m_Group>,
  HelpText<"Enable use of experimental RISC-V extensions.">;

Reply:
Oh sorry, I meant why `-menable-experimental-extensions` is in line 1620, not line 1619.

Reply:
Because I copied it from supm which has the same issue.